.github/workflows/backend_publish.yml

```diff
@@ -34,13 +35,11 @@ jobs:
 
     - name: Build and Publish Docker Image
       working-directory: backend
-      env:
```

Comment:
Tady bych nastavil             DOCKER_REGISTRY_USERNAME
a pod

Reply:
Proc? Neni pro to zadny duvod.

Reply:
you are right, i missed that that the gradle task is now used for build only